cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -0,0 +1,226 @@
+/* topdoc
+  name: Global Search molecule
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+          <div class="m-global-search">
+              <p class="short-desc">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/c21458b122409b41d50e3198afeff28f849777ae